fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeManager.java

```diff
@@ -277,16 +279,12 @@ private void initPrivilegeCollectionAllObjects(
 
     // called by initBuiltinRolesAndUsers()
     private RolePrivilegeCollection initBuiltinRoleUnlocked(long roleId, String name) {
-        if (!roleNameToId.containsKey(name)) {
-            RolePrivilegeCollection collection = new RolePrivilegeCollection(
-                    name, RolePrivilegeCollection.RoleFlags.MUTABLE);
-            roleIdToPrivilegeCollection.put(roleId, collection);
-            roleNameToId.put(name, roleId);
-            LOG.info("create built-in role {}[{}]", name, roleId);
-            return collection;
-        }
-        // public roles may be changed and persisted in image before restarted
-        return null;
+        RolePrivilegeCollection collection = new RolePrivilegeCollection(
```

Comment:
Always reinitiate the built-in role won't affect correctness, but i think the better way is to keep this fucntion and 
related condition unchanged, and load the info of the built-in role from image.